deepinv/physics/functional/radon.py

```diff
@@ -95,6 +129,15 @@ class Radon(nn.Module):
     :param int in_size: the size of the input image. If None, the size is inferred from the input image.
     :param torch.Tensor theta: the angles at which the Radon transform is computed. Default is torch.arange(180).
     :param bool circle: if True, the input image is assumed to be a circle. Default is False.
+    :param bool parallel_computation: if True, all projections are performed in parallel. Requires more memory but is faster on GPUs.
+    :param bool fan_beam: if True, use fan beam geometry, if False use parallel beam
+    :param dict fan_parameters: only used if fan_beam is True. Contains the parameters defining the scanning geometry. The dict should contain the keys
+        - "pixel_spacing" defining the distance between two pixels in the image, default: 0.077
+        - "source_radius" distance between the x-ray source and the rotation axis (middle of the image), default: 57.5
```

Comment:
this value is in pixels right?

Reply:
No, the default values are adapted from the scanner used in 
https://doi.org/10.5281/zenodo.8307932
where pixel spacing, source radius and detector radius where given in centimeter, I added this in the description. Actually, rescaling all the values (pixel spacing, source/detector radius, detector spacing) by the same constant does not change the operator. So, one could decide to fix the pixel spacing as 1 and give all the other values in pixels. However, if you have data from some scanner you always have to convert cm into pixels...

Reply:
ok, thanks for adding the explanation!